golden_response/staging/get_triples/place_type.json

```diff
@@ -3653,19 +3653,6 @@
       ],
       "provenanceId": "dc/5n63hr1"
     },
-    {
```

Comment:
https://datacommons.org/browser/country/ALA not much info. was removal intentional?

Reply:
This is from mid<->dcid mapping improvements. 

https://screenshot.googleplex.com/59c35064-5f10-451d-828b-25f2a1afdc32

This place is now mapped to https://datacommons.org/browser/nuts/FI2

Reply:
Ah ok. Thanks!